.circleci/config.yml

```diff
@@ -251,7 +251,7 @@ jobs:
   make-release:
     description: "Publishes the new version and creates a github release"
     macos:
-      xcode: 16.4.0
+      xcode: 13.4.0
```

Comment:
Hmm it's weird we need to downgrade to such an old version to run that job... What's the issue on the newer Xcode?